nodego/nodego.go

```diff
@@ -31,6 +32,38 @@ import (
 
 var fds = flag.String("fds", "", "fd1,fd2,...")
 
+var eventHandler http.Handler
+
+// UseEventHandler sets the event handler.
+func UseEventHandler(handler func(*FunctionEvent) error) {
```

Comment:
Would it be possible to have this be a piece of middeware instead?

Reply:
Sorry, to clarify:
This wouldn't necessarily be middleware in the traditional sense. Just instead have it return an http.HandlerFunc which the user manually registers. I like to keep control flow very clear when possible.

Reply:
Yes, that would probably be better. I sorta hacked it together and left it like this.

Reply:
I think I originally planned to actually subscribe to a pub sub topic for local testing, but found that it would make things too complicated.